### DISCUSSION TITLE
Failure in decoder conformance tests

### DISCUSSION THREAD

**Discussion by remul19** [fraunhoferhhi/vvdec#142] [Q&A] [answered]
Hi All,

I was trying out the decoder conformance test as per mentioned in [ https://github.com/fraunhoferhhi/vvdec/wiki/Build](url) 
And I used the below commands.
```
make release enable-bitstream-download=1
make test-all
```
But all test cases are failing and it seems like the test vectors (.bit files) are missing.
I am attaching a screenshot of the error message as a reference.
Can someone please guide me on how to resolve this issue ?

![image](https://github.com/fraunhoferhhi/vvdec/assets/136456486/8eb611c3-7c61-4c51-bcad-6dab14658e34)

**Comment by adamjw24** [comment, marked as answer]
Do `make distclean && make release enable-bitstream-download=1 && make test`

**Reply by remul19** [reply]
Thank you @adamjw24 ..
With your solution, all test cases passed.

![image](https://github.com/fraunhoferhhi/vvdec/assets/136456486/14c70221-7c1e-4d2c-9509-2b5678ed53ca)
